### ISSUE TITLE
i want to chanage pratmter in traing while resuming 

### ISSUE THREAD

**Issue by VYRION-Ai** [ultralytics/ultralytics#18233]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

i want to change parameter in training while resuming

`!yolo  train data=/content/data.yaml model=train2/weights/last.pt batch=12 patience=10  epochs=100 imgsz=512 cache=disk   `

i wan to add 

`!yolo  train data=/content/data.yaml model=train2/weights/last.pt batch=12 patience=10  epochs=50   mixup=0.2 shear=15 degrees=10 fliplr=0      imgsz=512 cache=disk   resume=True`

but it didnot work it resume using same old parameter

**Comment by UltralyticsAssistant** [comment]
👋 Hello @VYRION-Ai, thank you for your interest in Ultralytics 🚀! We recommend exploring our [Docs](https://docs.ultralytics.com) for answers to common questions and examples. For instance, the [Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) guide may provide valuable insights!

It looks like you're asking how to modify training parameters while resuming a run. If this is a ❓ Question, please provide additional context, such as whether you’re using a custom training setup or details on how the changes affect your workflow.

If this is a 🐛 Bug Report, it's crucial to share a [minimum reproducible example (MRE)](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us investigate the issue effectively. Please also specify your environment (Python and PyTorch versions, OS, etc.) and verify that you’re on the latest version of the `ultralytics` package:

```bash
pip install -U ultralytics
```

## Notes on Resuming Training

When resuming training, certain parameters may be locked to maintain consistency from the previously saved run. Parameters such as `epochs`, `mixup`, or `shear`, and others related to data augmentation might not be overridden unless you manually adjust the saved training run configuration.

For more details on training command behaviors, you can check our [CLI Tutorial](https://docs.ultralytics.com/usage/cli/) and [Training Modes](https://docs.ultralytics.com/modes/#train).

## Community Support

You can also reach out to the vibrant Ultralytics community:
- For real-time chat, join our [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions happen on [Discourse](https://community.ultralytics.com)
- Share your learnings or questions on our [Subreddit](https://reddit.com/r/Ultralytics)

## Supported Environments

To ensure compatibility, make sure you’re using a verified environment. YOLO is supported on platforms like:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Docker** environments. See instructions on running our [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Let us know if upgrading resolves your issue! This response is automated 🤖, but an Ultralytics engineer will assist you further as soon as possible.

**Comment by Y-T-G** [comment]
You can't change parameters while resuming
